nixos/modules/services/security/certmonger.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs, lib, config, ... }:
+with lib;
```

Comment:
Please avoid `with lib;` here, it's fine eg. in meta in packages.
https://discourse.nixos.org/t/new-nix-builtins-search/34059/

Reply:
done